packages/mysql/src/index.ts

```diff
@@ -239,26 +239,34 @@ function validateMySqlUrl(urlString: string) {
   }
 }
 
+function parseNullable<T, R>(value: T, parser: (value: T) => R): R | null {
+  if (value === null) {
```

Comment:
I've not seen `undefined` but it might be sensible to also check for `undefined` since that will also result in a `NaN` for the number parser.

```suggestion
  if (value === null || value === undefined) {
```